File: main/src/model/cardio_goal.java
package model;

public class cardio_goal extends Goal{
    // Intensity 1 - 5
    // 1 being least intensive
    // 5 being most intensive
    private int intense;
    private String activity;
    private int calorie_count;

    public cardio_goal() {

    }

    public void setActivity(String activity) {
        this.activity = activity;
    }

    private void set_intensity(int a) {
        this.intense = a;
    }

    public void setCalorie_count(int calorie_count) {
        this.calorie_count = calorie_count;
    }
}

File: sustainagoal/src/model/cardioGoal.ts
import Goal from "./Goal";

export default class cardioGoal extends Goal{
    // Intensity 1 - 5
    // 1 being least intensive
    // 5 being most intensive
    private readonly goalCalorie: number = 0;
    private  intense: number = 0;
    private  activity: string = "";
    // private  calorie_count: number = 0;

    constructor(goalAmount: number, intense: number, activity: string) {
       super();
       this.intense = intense;
       this.activity = activity;
       this.goalCalorie = goalAmount;
    }

    setIntense(intensity: number) {
        this.intense = intensity;
    }

    setActivity(activity: string) {
        this.activity = activity;
    }

    // setCalorieCount(cc: number) {
    //     this.calorie_count = cc;
    // }

    getGoalCalorie(): number {
        return this.goalCalorie;
    }

    getActivity(): string {
        return this.activity;
    }

    getIntensity(): number {
        return this.intense;
    }

    // getCurrentCalorie(): number {
    //     return this.calorie_count;
    // }
}